examples/spark2/src/main/scala/org/apache/carbondata/benchmark/ConcurrencyQueryBenchmark.scala

```diff
@@ -15,47 +15,63 @@
  * limitations under the License.
  */
 
-package org.apache.carbondata.examples
+package org.apache.carbondata.benchmark
 
 import java.io.File
+import java.text.SimpleDateFormat
 import java.util
+import java.util.Date
 import java.util.concurrent.{Callable, Executors, Future, TimeUnit}
 
 import scala.util.Random
 
 import org.apache.spark.sql.{DataFrame, Row, SaveMode, SparkSession}
 import org.apache.spark.sql.types._
 
-import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.core.constants.{CarbonCommonConstants, CarbonVersionConstants}
 import org.apache.carbondata.core.util.{CarbonProperties, CarbonUtil}
 
 // scalastyle:off println
-object ConcurrencyTest {
-
-  var totalNum = 100 * 1000 * 1000
-  var ThreadNum = 16
-  var TaskNum = 100
-  var ResultIsEmpty = true
-  val cardinalityId = 10000 * 10000
+object ConcurrencyQueryBenchmark {
```

Comment:
Change to ConcurrentQueryBenchmark

Reply:
Can you explain in comment what will print out in this benchmark

Reply:
ok, I add some annotation for ConcurrentQueryBenchmark